static/src/javascripts/projects/commercial-control/modules/check-dispatcher.js

```diff
@@ -2,15 +2,11 @@
 import config from 'lib/config';
 import { allEmailCanRun, listCanRun } from 'common/modules/email/run-checks';
 import { getListConfigs } from 'common/modules/email/email-article';
-import {
-    emailTests,
-    contributionsTests,
-    userIsInAClashingAbTest,
-} from 'common/modules/experiments/ab-test-clash';
 import { trackAdRender } from 'commercial-control/modules/dfp/track-ad-render';
 import { commercialFeatures } from 'common/modules/commercial/commercial-features';
 import { checks } from 'common/modules/check-mediator-checks';
 import { resolveCheck, waitForCheck } from 'common/modules/check-mediator';
+import { getTest as getAcquisitionTest } from 'common/modules/experiments/acquisition-test-selector';
```

Comment:
thanks for fixing up `commercial-control` too. It's used by a 1% group of the audience, so it is worth copying your fixes too. It's kind of like back-porting your fix. Ordinarily, by fixing just files in `commercial` folder, you are fixing behaviour for 99% of the audience.

Reply:
definite overuse of the word fix.